content/sphinx.md

```diff
@@ -357,30 +357,25 @@ def multiply(a: float, b: float) -> float:
     return a * b
 ```
 
-2. Add a file `api.md` in the same folder as `index.rst`, with the following content:
-````md
-# API reference
-
-## example
+2. In the file `conf.py` modify "extensions" and add 3 lines:
+```python
+extensions = ['myst_parser', "sphinx.ext.autodoc"]
 
-```{eval-rst}
-.. automodule:: example
-   :members:
+autodoc2_packages = [
+    "multiply.py"
+]
 ```
```

Comment:
This is more similar to [autosummary](https://www.sphinx-doc.org/en/master/usage/extensions/autosummary.html), but this approach is more practical rather than listing modules one-by-one, IMHO. :+1: 

In case if we need to recommend something simpler we can show [`autodoc2-object` ](https://sphinx-autodoc2.readthedocs.io/en/latest/autodoc.html)which is the MyST friendly equivalent of `.. automodule` rST directive

Reply:
I will check it out, thanks!

Reply:
I agree, I think this is better to teach first. We just list the package and it finds everything for us.